### DISCUSSION TITLE
Cannot setup locally hosted instance

### DISCUSSION THREAD

**Discussion by amzon-ex** [vrtmrz/obsidian-livesync#540] [Q&A]
Hi! I'm following your instructions at https://github.com/vrtmrz/obsidian-livesync/blob/main/docs/setup_own_server.md word-by-word to locally host a couchdb instance and sync my Obsidian vault to it. I'm not going to expose my couchdb instance to the internet. I'm running couchdb on docker.

I keep getting an error saying 
```
could not connect to http://192.168.xx.xx:5984/obsidiannotes: Name or password is incorrect.
```

I don't understand what is going wrong here. I can access http://192.168.xx.xx:5984/ through a browser using the admin password.

**Comment by spcano01** [comment]
I use a domain, but when I was testing locally - don't think you need the /obsidiannotes path at the end.  Don't have anything beyond https://couchdb.domain.tld in my URI field either.

**Reply by amzon-ex** [reply]
Hi, thanks for your reply! 

The error message that I posted is generated by the plugin itself in Obsidian - obsidiannotes is the name of the database that is required when you run `deno` to obtain the setup URI. I didn't choose to add anything by myself, this is just the default instructions at play. MAybe it's the addresses somehow?
